replication-commands/src/main/java/org/codice/ditto/replication/commands/ConfigAddCommand.java

```diff
@@ -133,4 +162,20 @@ protected final Object executeWithSubject() {
 
     return null;
   }
+
+  private ReplicationSite getOrCreateSite(String site) throws MalformedURLException {
+    ReplicationSite replicationSite =
+        siteStore
+            .getSites()
+            .stream()
+            .filter(s -> s.getUrl().toString().equals(site) || s.getName().equals(site))
```

Comment:
❓ Are we allowing them to specify the site by name OR url? It only says that the command takes a destination specified by a URL in the description. If the user does specify a name and it doesn't match an existing site then it looks like this will print out a message saying that it was given an invalid URL.

Reply:
I was thinking this might be useful but you are right. Removing the name check.